tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/RecordWithClosureTest.java

```diff
@@ -37,7 +37,7 @@ public void setup() {
         compileResult = BCompileUtil.compile("test-src/record/record_closure_default.bal");
     }
 
-    @Test(description = "Test record type with closure variable in default value")
+    @Test(description = "Test record type with closure variable in default value", enabled = false)
```

Comment:
The test disabled is https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/record/record_closure_default.bal#L1.

With the requirement that record field defaults have to be isolated, based on the current implementation I don't think we can test what we originally tested here.

But disabled this without removing it, since we may be able to add back a different version of the test once we support isolated module level variables (https://github.com/ballerina-platform/ballerina-lang/issues/25898).